packages/client/src/testHelpers/Renderer.js

```diff
@@ -187,9 +193,11 @@ export default class Renderer {
     return mount(
       this.withApollo(
         clientModules.getWrappedRoot(
-          <Router history={this.history}>
-            <Switch>{clientModules.routes}</Switch>
-          </Router>
+          <I18nextProvider i18n={clientModules.i18n}>
```

Comment:
@alekseyzadorozhniy Do you see `clientModules.getWrappedRoot` one line above?

Reply:
Oh, I think I understand what you mean